### PR TITLE
Add Cloud Shell tutorial

### PR DESCRIPTION
**Pull request by kelsk**
This Cloud Shell tutorial provides simple deployment instructions setting up either a minikube or GKE cluster.

Test the tutorial using the link below:

[Open in Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/kelsk/microservices-demo&cloudshell_workspace=.&cloudshell_tutorial=docs/cloudshell-tutorial.md&cloudshell_git_branch=kelsk-cloudshell-tutorial)

### CODE REVIEW

File: docs/cloudshell-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+Minikube creates a local Kubernetes cluster on Cloud Shell.
+
+1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">minikube</walkthrough-editor-spotlight> on the status bar located at the bottom of the editor window.
+
+2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**. 
+
+3. If prompted, authorize Cloud Shell to make a GCP API call with your credentials.
+
+Once minikube has finished starting, you're ready to move on to the next step.
```
Comment (peterlin741) [inline docs/cloudshell-tutorial.md:29]: nit: do you want to mention that starting minikube may take a few minutes?

File: docs/cloudshell-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+Set up a Kubernetes cluster using the instructions below for either **minikube** or **GKE**.
+
+---
+### Minikube instructions
+
+Minikube creates a local Kubernetes cluster on Cloud Shell.
+
+1. Click <walkthrough-editor-spotlight spotlightId="minikube-status-bar">minikube</walkthrough-editor-spotlight> on the status bar located at the bottom of the editor window.
+
+2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start**.
```
Comment (peterlin741) [inline docs/cloudshell-tutorial.md:25]:
```suggestion
2. The command palette will prompt you to choose which minikube cluster to control. Select **minikube** and, in the next prompt, click **Start** if the cluster has not already been started. 
```

File: docs/cloudshell-tutorial.md (hunk 1 above)
Comment (kelsk) [reply docs/cloudshell-tutorial.md:29]: Good idea - added.

**Review by askmeegs — APPROVED** [review]
Thank you for this @kelsk !! Tested in Cloud Shell with minikube. Cloud Code working, all spotlights + hyperlinks working. 

Screen Shot 2021-07-14 at 2 55 39 PM

**Pull request merged**